public/leaderboard.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
```

Comment:
Protip: make this leaderboard/index.html & you won’t need the rewrite!

Reply:
ahhh i thought there was a way